tests/test_livysession.py

```diff
@@ -39,11 +36,18 @@ class TestLivySession(object):
     log_json = json.loads('{"id":6,"from":0,"total":212,"log":["hi","hi"]}')
 
     def __init__(self):
+        self.http_client = None
```

Comment:
By creating test fixture before executing each test case.
Setting _multiprocess_can_split_ = True in the test class will do the trick.